tests/data/config/config.yaml

```diff
@@ -171,4 +175,28 @@ SageMaker:
   EdgePackagingJob:
     OutputConfig:
       KmsKeyId: 'kmskeyid1'
-    RoleArn: 'arn:aws:iam::555555555555:role/IMRole'
\ No newline at end of file
+    RoleArn: 'arn:aws:iam::555555555555:role/IMRole'
+    ResourceKey: 'kmskeyid1'
+  PythonSDK:
```

Comment:
Why this is needed here? 
This is already in the file from line 3:28.

Reply:
Thanks - Yes. it is not needed, got incorrectly added as part of a git merge conflict resolve. Will remove it in the next commit